utilities/utils.py

```
@@ -106,6 +107,61 @@ def _fetch_and_store_cacert(
+                continue
+
+            if hasattr(csv, "spec") and csv.spec:
+                display_name = getattr(csv.spec, "displayName", None)
+                mtv_version_str = getattr(csv.spec, "version", None)
+
+                if display_name and display_name == "Migration Toolkit for Virtualization Operator":
+                    mtv_version = mtv_version_str
+                    break
+
+    except Exception as e:
+        raise RuntimeError(f"Failed to determine MTV version: {e}") from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)